src/Explorer/Notebook/NotebookContainerClient.ts

```diff
@@ -97,14 +93,30 @@ export class NotebookContainerClient {
         const connectionStatus: ContainerConnectionInfo = {
           status: ConnectionStatusType.Failed,
         };
-        useNotebook.getState().resetConatinerConnection(connectionStatus);
-        useNotebook.getState().setIsRefreshed(true);
+        useNotebook.getState().resetContainerConnection(connectionStatus);
```

Comment:
>resetContainerConnection [](http://example.com/codeflow?start=31&length=24)

please think about this. 

i think this is too aggressive and can cause very bad ux experience. if the api had bad response once, could the ws connection still be fine..


should evaluate this place with auto reconnect.